tests/test_chat.py

```diff
@@ -881,6 +896,128 @@ async def make_assertion(*_, **kwargs):
         monkeypatch.setattr(chat.get_bot(), "decline_chat_join_request", make_assertion)
         assert await chat.decline_join_request(user_id=42)
 
+    async def test_create_forum_topic(self, monkeypatch, chat):
+        async def make_assertion(*_, **kwargs):
+            return (
+                kwargs["chat_id"] == chat.id
+                and kwargs["name"] == "New Name"
+                and kwargs["icon_color"] == 0x6FB9F0
+                and kwargs["icon_custom_emoji_id"] == "12345"
```

Comment:
we could use a fixture that returns a real emoji ID, I have one in `test_forum.py`. But maybe it's not needed here.

Same for similar lines in other test functions below

Reply:
we just check if the params are being passed to the method, we don't do any checks for the correct-ness of the value